.github/pull_request_template.md

```diff
@@ -14,4 +14,6 @@ Checklist:
 * [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
 * [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
 * [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)). 
+* [ ] My new feature complies with the [feature status](https://github.com/argoproj/argoproj/blob/master/community/feature-status.md) guidelines.
 
```

Comment:
Can we add something like "I have added a brief description of why this PR is necessary and/or what this PR solves"? I've seen PRs with no description and/or maybe a few words which doesn't give insights as to why it is needed.